src/operator/nn/activation.cu

```diff
@@ -31,39 +31,73 @@
 
 namespace mxnet {
 namespace op {
+
+#if MXNET_USE_CUDNN == 1
+
+template<typename DType>
+static CuDNNActivationOp<DType> &get_cudnn_op(const ActivationParam& param) {
+#if DMLC_CXX11_THREAD_LOCAL
+  static thread_local CuDNNActivationOp<DType> cudnn_op;
+#else
+  static MX_THREAD_LOCAL CuDNNActivationOp<DType> cudnn_op;
```

Comment:
we can probably drop support for non DMLC_CXX11_THREAD_LOCAL. I think it only affects vs 13?

Reply:
it won't pass the tests in github. i guess we can do it after we upgrade the tests?